packages/react-ai/src/hooks/shared.ts

```diff
@@ -2,7 +2,11 @@ import { DataState } from '@aws-amplify/ui-react-core';
 import { GraphQLFormattedError } from '../types';
 
 export type DataClientState<T> = Omit<DataState<T>, 'message'> & {
+  /**
+   * @deprecated - Please use `errors` instead
+   */
   messages?: GraphQLFormattedError[];
+  errors?: GraphQLFormattedError[];
```

Comment:
Prefer removing the dash here, it adds an additional in the type preview:

![CleanShot 2025-04-02 at 09 20 25@2x](https://github.com/user-attachments/assets/b86c1762-a4d1-40c6-bfa0-a9139c6b2821)

```suggestion
  /**
   * @deprecated will be removed in a future major version. Superseded by `errors`
   * @description errors returned from the websocket connection
   */
  messages?: GraphQLFormattedError[];
  /**
   * @description errors returned from the websocket connection
   */
  errors?: GraphQLFormattedError[];
```